stablehlo/transforms/Passes.td

```diff
@@ -178,3 +178,15 @@ def StablehloLegalizeCompositeToCallPass :
     ListOption<"exceptListOption", "except", "std::string", "Names of composites that should not be replaced with calls.">
   ];
 }
+
+def StablehloLegalizeQuantToIntPass : Pass<"stablehlo-legalize-quant-to-int", "mlir::func::FuncOp"> {
+  let summary = "Convert from StableHLO quantized ops to StableHLO primitive ops.";
+
+  let description = [{
+    Convert from StableHLO quantized ops with StableHLO quant types to StableHLO primitive ops
+    like int ops.
```

Comment:
"Convert StableHLO programs using UniformQuantized types the semantically equivalent integer math."

I feel like when people talk about this pass they refer to "int math" so would like to have it in the description for clarity.

Reply:
done.